doc/changelog.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+- :release:`0.5.0 <2020.05.27>`
+
+- :feature:`0` Python 3.8 is now supported and tested.
```

Comment:
Also, we drop support for Python 2.7. I don't know if there's a mark for breaking changes.

Reply:
The PR is https://github.com/pygae/galgebra/pull/232

Reply:
I think that appears in the [Support] section of the notes already.

Reply:
Nope, you're right, that's missing.
```suggestion
- :feature:`0` Python 3.8 is now supported and tested. Python 2.7 is no longer supported.
```

Let's just put them in the same bullet point so that users can find it easily.

Reply:
```suggestion
- :feature:`232` Python 3.8 is now supported and tested.
- :feature:`212` The support for Python 2.7 is dropped and more features from Python 3.x are used.
```

Reply:
The above is my story to make dropping 2.7 a feature!

Reply:
```suggestion
- :feature:`232` Python 3.8 is now supported and tested.
- :feature:`212` Python 2.7 is no longer supported, allowing more features from Python 3.x to be used.
```